### ISSUE TITLE
Add MARC 21 to "Related standards"

### ISSUE THREAD

**Issue by JordanLyons** [openreferral/specification#200]
The Library of Congress has a [format called MARC 21](https://www.loc.gov/marc/community/) for community information that should maybe be included in the "Related standards" section. It was referenced in a library science article I read about information and referral.

**Comment by NeilMcKechnie** [comment]
Hey this is pretty interesting @JordanLyons . Have you looked closely to see to what degree it has a similar data structure and elements as HSDS?

**Comment by JordanLyons** [comment]
I found out about it in this academic paper that compares it to the AIRS format: https://www-emeraldinsight-com.weblib.lib.umt.edu:2443/doi/full/10.1108/JD-10-2014-0141

**Comment by kathryn-ods** [comment]
I agree this would be sensible to add to the related standards section. I propose adding this 

**MARC 21 Format for Community Information**
MARC 21 (machine-readable cataloging) is a standard set of digital formats for the machine-readable description of items catalogued by libraries. 

The MARC 21 Format for Community Information is used for MARC records describing a service-providing agency, such as a local homeless shelter or tax assistance provider. 

It includes 5 subcategories:
- Individual - an individual with a particular expertise
- Organization - an organization or any group
- Program or service - an offering or activity of a group or institution which carries out the purposes of that group or institution
- Event - a scheduled happening
- Other - a kind of Community Information not mentioned above. For example, the data may be for a facility, such as a planetarium on a university campus.

**Comment by kathryn-ods** [comment]
additionally - I found this example data which is a useful reference for what these sources look like https://www.itsmarc.com/crs/MergedProjects/CommInfo/comminfo/appendix_b_record_examples.htm

it's interesting how flexible this standard seems to be in some aspects e.g. [representing schedules](https://www.loc.gov/marc/community/concise/ci307.html) maybe this is because the standard is designed for the specific case of librarians sharing information with individual service users? 

I don't know how well used this format is for community information today - apparently it's still widely used by libraries that may be more for books.
